From: Bart Van Assche <bart.vanassche@sandisk.com>
Date: Tue, 14 Jun 2016 17:03:13 +0200
Subject: block/partition-generic.c: Remove a set-but-not-used variable
References: bnc#1003941,FATE#321732
Patch-Mainline: v4.8-rc1
Git-commit: aa8d15bfe4ed23b97bf3990e2901844efc6347ec

A value is assigned to the variable 'info' but that value is never
used. Hence remove the variable 'info'.

Signed-off-by: Bart Van Assche <bart.vanassche@sandisk.com>
Signed-off-by: Jens Axboe <axboe@fb.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 block/partition-generic.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/block/partition-generic.c b/block/partition-generic.c
index 84c53f0..ab3d01d 100644
--- a/block/partition-generic.c
+++ b/block/partition-generic.c
@@ -495,7 +495,6 @@ rescan:
 	/* add partitions */
 	for (p = 1; p < state->limit; p++) {
 		sector_t size, from;
-		struct partition_meta_info *info = NULL;
 
 		size = state->parts[p].size;
 		if (!size)
@@ -530,8 +529,6 @@ rescan:
 			}
 		}
 
-		if (state->parts[p].has_info)
-			info = &state->parts[p].info;
 		part = add_partition(disk, p, from, size,
 				     state->parts[p].flags,
 				     &state->parts[p].info);
-- 
1.8.5.6

